demo/index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<!--
+    @license
+    Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
+    This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+    The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+    The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+    Code distributed by Google as part of the polymer project is also
+    subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Core Label</title>
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <link rel="import" href="../iron-label.html">
+  <link rel="import" href="../../paper-checkbox/paper-checkbox.html">
+  <link rel="import" href="../../paper-slider/paper-slider.html">
+  <link rel="import" href="../../paper-button/paper-button.html">
+</head>
+<body>
+
+  <label>regular label<input type="checkbox"></label>
```

Comment:
Would you mind prettifying the demo with one of the things from `paper-styles/demo-pages.css`? You can look at `paper-checkbox` for a horizontal layout, or `paper-input` for a vertical one